src/subscription-user/subscription-user.controller.ts

```diff
@@ -14,6 +16,33 @@ import { SubscriptionUserService } from './subscription-user.service';
 export class SubscriptionUserController {
   constructor(private readonly subscriptionUserService: SubscriptionUserService) {}
 
+  @Get('/subscription_user')
```

Comment:
I think this endpoint doesn't need to be GET `api/v1/subscription_user/subscription_user`.  It could just be left empty as some of our other module controllers and be GET `api/v1/subscription_user`. Does that make sense?

Reply:
Yes. Good catch.